### PR TITLE
cloud-init/daily_deb.sh: use ctool.

### PR DESCRIPTION
**Pull request by smoser**
ctool provides much of the functionality that was used here.
Use it to make things easier.

### CODE REVIEW

**Comment by smoser** [comment]
This will now have output like this:

    executing: ctool run-container --verbose --destroy --name=cloud-init-ubuntu-daily-ppa-99b97790 --as-root --artifacts=. --copy-out=download/ -- ubuntu-daily:cosmic bash -s download <'./cloud-init/daily_deb.sh'
    Waiting for boot of container 'cloud-init-ubuntu-daily-ppa-99b97790'.
    ....[cloud-init-ubuntu-daily-ppa-99b97790] boot marked failed after check number 4.
    cloud-init-ubuntu-daily-ppa-99b97790 did not finish booting. ret=1
    deleting container 'cloud-init-ubuntu-daily-ppa-99b97790' due to --destroy
    failed executing ctool run-container --verbose --destroy --name=cloud-init-ubuntu-daily-ppa-99b97790 --as-root --artifacts=. --copy-out=download/ -- ubuntu-daily:cosmic bash -s download < './cloud-init/daily_deb.sh'

^^ that is the failed path that shows 'boot marked failed'.

The happy path looks like: http://paste.ubuntu.com/p/Yd8BcQSFdy/

File: cloud-init/daily_deb.sh [hunk 1]
```diff
@@ -104,39 +56,28 @@ main() {
...
-    file=$(inside "$name" ls /root/) ||
-        fail "failed: ls"
-
-    debug 1 "pulling cloud-init deb"
-    lxc file pull "$name"/root/"$file" . ||
-        fail "failed: pull file"
-
-    echo "Download successful!"
-
+    release=5ee1af40be67
```
Comment (powersj) [inline cloud-init/daily_deb.sh]: I assume you were testing with a working cosmic image, but should this be hardcoded?

File: cloud-init/daily_deb.sh [hunk 2]
```diff
@@ -104,39 +56,28 @@ main() {
...
+    rm -Rf download || { error "failed removing download/"; return 1; }
+    set -- \
+        ctool run-container --verbose --destroy "--name=$name" \
```
Comment (powersj) [inline cloud-init/daily_deb.sh]: Where is ctool?

Comment (smoser) [reply cloud-init/daily_deb.sh]: https://github.com/CanonicalLtd/uss-tableflip/blob/master/scripts/ctool

i'd like to start assuming some things present. just as reasonable building blocks.

File: cloud-init/daily_deb.sh (hunk 1 above)
Comment (smoser) [reply cloud-init/daily_deb.sh]: yeah, and removed.

File: cloud-init/daily_deb.sh (hunk 2 above)
Comment (smoser) [reply cloud-init/daily_deb.sh]: updated to download ctool.

**Comment by smoser** [comment]
updated to download ctool from github before using.

**Pull request merged**